src/tools/pontoon/translation_workspace.md

```diff
@@ -176,6 +176,7 @@ The space below the editor displays the list of [translations](glossary.md#trans
 
 Each entry contains:
 * The name of the translator and their profile picture (linking to their profile).
+* The status of the user (see [Status banners](http://localhost:3000/tools/pontoon/translation_workspace.html#status-banners)).
```

Comment:
```suggestion
* The status of the user (see [Status banners](#status-banners)).
```

Reply:
Yikes sorry for leaving the `localhost:3000` part in :P